workshops/portfolio/README.md

```diff
@@ -0,0 +1,563 @@
+# Building Your First Website
+
+## Our Mission
+
+Our mission:
+
+- make your personal website
+- put it on the internet for the whole world to see!
+
+Your final design looks like this:
+
+> ![](img/final_1.png)
+
+Here is a link to a [live demo](http://output.jsbin.com/fugoki/2)
+
+And here is the [final code](http://jsbin.com/fugoki/2/edit) for the live demo.
+
+<!-- TODO: Add extra content -->
+
+<!-- After club, you can go home and try to make your website look like this:
+
+<!-- TODO -->
+
+To do this, you will be learning the basics of two languages: HTML and CSS.
+
+Every website that you have ever seen are written in these two languages.
+
+## Microsoft Word for Code
+
+Just like you use Microsoft to write English, you can use JSBin to write code.
+
+Open [http://jsbin.com](http://jsbin.com).
+
+Go ahead and close the popup:
+
+> ![](img/close.gif)
+
+Then delete everything on the left. We don't need it right now.
+
+> ![](img/delete.gif)
+
+Then write your name in the text box. My name is `Jonathan Leung` so that's
+what I will write:
+
+> ![](img/jonathan_leung.gif)
+
+Notice that that what you write on the left side of the screen is reflected
+on the right.
+
+> ![](img/left.png)
+
+Note that we are writing "HTML" code.
+
+> ![](img/right.png)
+
+<!-- Original image: ![](img/original.png) -->
```

Comment:
Yes, I added this as a clarifying note:

```html
<!-- This is a link to the original image in case any changes
     want to be made in the future: ![](img/original.png) -->
```